client/client.go

```diff
@@ -10,10 +10,10 @@ import (
 )
 
 type UpgradeChecker struct {
-	Address          string
-	UpgradeRequester UpgradeRequester
-	queryPeriod      time.Duration
-	stopCh           chan struct{}
+	Address                string
+	UpgradeRequester       UpgradeRequester
+	InitialRequestInterval time.Duration
```

Comment:
Rename it to `DefaultRequestInterval`. Since it will be used in case we don't receive `requestInterval` from the server.

Reply:
Thank you. Fixed